tooling/nargo_fmt/tests/expected/tuple.nr

```diff
@@ -10,7 +10,9 @@ fn main() {
     (/*test*/ 1,);
     (/*a*/ 1/*b*/,);
     (/*a*/ 1/*b*/, /*c*/ 2/*d*/, /*c*/ 2/*d*/);
-    (/*a*/ 1/*b*/, /*c*/ 2/*d*/, /*c*/ 2/*d*/, /*e*/ 3/*f*/);
+    (
+        /*a*/ 1/*b*/, /*c*/ 2/*d*/, /*c*/ 2/*d*/, /*e*/ 3/*f*/
+    );
```

Comment:
This one seems odd to me. Wouldn't it be shorter if it were all on the same line?